src/tooltip.js

```diff
@@ -233,6 +246,7 @@ angular.module('ts.utils')
           if($scope.tooltipScope)
             $scope.tooltipScope.$destroy();
           angular.element(arrowBoxContainer).remove();
+          angular.element($window).off('resize', positionTooltip);
```

Comment:
@dozoisch Window listener to be turned off upon tooltip being destroyed. Would this be the right place?